src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -99,6 +112,8 @@ public class PullRequestHandler : IHttpHandler
             {
                 case "master":
                 case "origin/master":
+                case "origin/release/1.0.0-rc2":
+                case "origin/infrastructure":
```

Comment:
Why don't we just open it up for all branches and let it fail if a branch does not exist? I really want to minimize the churn to PRService.